src/interpolation_functions.jl

```diff
@@ -0,0 +1,304 @@
+using ApproxFun:
+  Interval, Fourier, Chebyshev, Fun, ProductFun, coefficients, chop, ncoefficients
+using Interpolations: LinearInterpolation, Line
+
+""" Helper function for fourier_itensornetwork and fourier_2D_itensornetwork """
+function fourier_term(s::IndsNetworkMap, j::Integer, d::Integer)
+  if j == 1
+    return const_itn(s)
```

Comment:
I think this should pass the dimension too

Reply:
const_itn doesn't take dimension as an argument so I can't

Reply:
You could try to get a delta_kernel working or modify `const_itn`